obstacle_stop_planner_refine/test/src/test_obstacle_stop_planner.cpp

```diff
@@ -27,6 +27,34 @@ using sensor_msgs::msg::PointCloud2;
 using autoware_planning_msgs::msg::Trajectory;
 using namespace std::chrono_literals;
 
+// FIXME: Rename function name
+Trajectory convertPointsToTrajectoryWithYaw2(
```

Comment:
What is Yaw**2**?

Reply:
This is a temporary name and will be changed later.